index.js

```diff
@@ -49,67 +50,68 @@ client.on("message", function (message) {
     inventoryCommand(message);
   } else if (message.content.startsWith("!lookup")) {
     lookUpCommand(message);
+  } else if (message.content.startsWith("!remove")) {
+    removeCardCommand(message);
   }
 });
 
 const drawCommand = (message) => {
-  var ref = db.ref("users");
+  const ref = db.ref("users");
   ref.once("value", function (snapshot) {
     const userData = snapshot.val();
+    let userID;
+    let userInventory;
     let userDraw = message.author.id;
     let user = userData[userDraw];
     const userDoesNotExist = !user;
     let timeDifference;
     if (user) {
       let lastBotUse = user.lastDrawTime;
       timeDifference = Date.now() - lastBotUse;
-      console.log(userData);
-      console.log(lastBotUse);
-      console.log(Date.now());
-      console.log(timeDifference);
+      userID = message.author.id;
+      userInventory = userData[userID].inventory;
```

Comment:
i.e. you can do the inventory size check here and send the "remove cards" message if the inventory limit is hit and then shortcircuit the rest of this function by doing a `return`.